data/tasks.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tasks>
+    <task>
+        <id>1</id>
+        <task>COMP1234 assignment</task>
```

Comment:
should i change property to name?

Reply:
Ya change it to `description` or `title`